README.md

```diff
@@ -6,6 +6,14 @@ ElasticsearchODBC is a read-only ODBC driver for Windows and Mac for connecting
 
 The driver is compatible with ODBC 3.51.
 
+## Supported Versions
+
+
+  | Operating System  | Version |
+  | ------------- |-------------|
+  |  Windows    |  Windows 10 |
+  |  MacOS    |  Mojave 10.14.6 |
```

Comment:
This has worked for us on Catalina (10.15.4), so let's include that as well

Reply:
Thanks. Added.